.github/workflows/build-jp5.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: setup workspace
-      run: ./setup_workspace.sh 5.0.2
+      run: yes | ./setup_workspace.sh 5.0.2
```

Comment:
What does `yes` means? :)

Reply:
I suggest to run it on any branch.
```
on:
  push:
    branches: ['**']
  pull_request:
    branches: ['**']
```

Reply:
> What does `yes` means? :)

answer "yes" for all questions. In that case it accepts nvidia license